<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KeyReader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_show_image">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>901</width>
      <height>661</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>实时影像</string>
    </property>
    <widget class="QLabel" name="label_show_image">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>29</y>
       <width>851</width>
       <height>621</height>
      </rect>
     </property>
     <property name="text">
      <string>实时影像</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_result">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>901</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>结果展示区</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_key_id">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>220</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_key_code">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>580</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>72</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>产品信息</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>182</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_plant">
          <property name="text">
           <string>厂家:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_plant">
          <property name="minimumSize">
           <size>
            <width>139</width>
            <height>39</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>139</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_product">
          <property name="text">
           <string>产品：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_product">
          <property name="maximumSize">
           <size>
            <width>139</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_marble_number">
          <property name="text">
           <string>弹子数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_marble_number">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_single_row">
          <property name="text">
           <string>单排齿</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_double_row">
          <property name="text">
           <string>双排齿</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_change_product">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>187</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择产品：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>860</y>
      <width>901</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>189</width>
      <height>225</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_set_calibration_line">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 131, 131);</string>
       </property>
       <property name="text">
        <string>设定标定线</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_self_calibrate">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 131, 131);</string>
       </property>
       <property name="text">
        <string>校准(自动调参)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_manual_calibrate">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 131, 131);</string>
       </property>
       <property name="text">
        <string>手动调参</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_check_image">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 131, 131);</string>
       </property>
       <property name="text">
        <string>查看图像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(131, 131, 131);</string>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1229</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_check_image</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>capture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_manual_calibrate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>manual_adjustment_parameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_self_calibrate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>self_calibration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_set_calibration_line</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_calibration_line()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_change_product</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>change_product()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>capture()</slot>
  <slot>start()</slot>
  <slot>manual_adjustment_parameters()</slot>
  <slot>self_calibration()</slot>
  <slot>set_calibration_line()</slot>
  <slot>change_product()</slot>
 </slots>
</ui>
